datasets/ONS-UK-Trade-in-goods-CPA-08/info.json

```diff
@@ -20,23 +21,28 @@
         "main_issue": 15,
         "columns": {
             "Value": {
+                "type": "observations",
                 "unit": "http://gss-data.org.uk/def/concept/measurement-units/gbp-million",
-                "measure": "http://gss-data.org.uk/def/trade/measure/trade-in-goods"
+                "measure": "http://gss-data.org.uk/def/trade/measure/trade-in-goods",
+                "datatype": "integer"
             },
             "CDID": {
-                "dimension": "http://gss-data.org.uk/def/trade/property/dimension/cdid",
-                "value": "http://gss-data.org.uk/def/trade/concept/cdid/{cdid}"
+                "type": "dimension",
+                "uri": "http://gss-data.org.uk/def/trade/property/dimension/cdid",
+                "value": "http://gss-data.org.uk/def/trade/concept/cdid/{+cdid}"
             },
             "Flow Direction": {
-                "dimension": "http://gss-data.org.uk/def/trade/property/dimension/flow-directions",
-                "value": "http://gss-data.org.uk/def/trade/concept/flow-directions/{flow_direction}"
+                "type":"dimension",
+                "uri": "http://gss-data.org.uk/def/trade/property/dimension/flow-directions",
+                "value": "http://gss-data.org.uk/def/trade/concept/flow-directions/{+flow_direction}"
             },
             "Period": {
-                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+                "type": "dimension",
+                "uri": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
                 "value": "http://reference.data.gov.uk/id/{+period}"
             },
             "Product": {
-                "parent": "http://gss-data.org.uk/def/trade/property/dimension/product"
+                "type": "dimension"
             }
         }
     },
```

Comment:
You may delete these lines as we don't want dataset to be published before we take a look
    "load": {
        "publish": true
    },